packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,257 @@
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">buttonIconPosition <code>enum</code></dt>
+    <dd class="dummy-indent">
+      <p>
+        Positions the icon before or after the text. Exists if
+        <code class="dummy-code">hasButton=true</code>
+        and if
+        <code class="dummy-code">buttonText</code>
+        exists.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">
+          leading
+        </li>
+        <li>
+          trailing
+        </li>
+      </ol>
+    </dd>
+
+    <dt>hasLinkStandalone <code>boolean</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>true</li>
+        <li class="default">false</li>
+      </ol>
+    </dd>
+
+    <dt class="dummy-indent">linkStandaloneIcon <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show an icon on the Link (Standalone). Exists if
+        <code class="dummy-code">hasLinkStandalone=true</code>.
+      </p>
+      <p>Acceptable value:</p>
+      <p>Any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight icon</a>
+        name.
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">linkStandaloneText <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show text on the Link (Standalone). Exists if
+        <code class="dummy-code">hasLinkStandalone=true</code>.
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">linkStandaloneIconPosition <code>enum</code></dt>
+    <dd class="dummy-indent">
+      <p>
+        Positions the icon before or after the text. Exists if
+        <code class="dummy-code">haslinkStandalone=true</code>
+        and if
+        <code class="dummy-code">linkStandaloneText</code>
+        exists.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">
+          leading
+        </li>
+        <li>
+          trailing
+        </li>
+      </ol>
+    </dd>
+
+    <dt>hasDismiss <code>boolean</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">true</li>
+        <li>false</li>
+      </ol>
+    </dd>
+
+    <dt>“splattributes”</dt>
+    <dd>
+      <p><code>...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3">Scratch showcase, static markup of global warning</h3>
+
+  <Hds::Alert />
+
+  <p>
+    TODO next steps:
+  </p>
+  <ul>
+    <li>Dynamic variant global vs page/inline w/test. Move parent CSS to JS</li>
+    <li>Dynamic just icon and title w/test</li>
+    <li>Footer variants: Dynamic button and/or link (standalone) w/test</li>
+    <li>Add dismiss w/test</li>
+    <li>Style variant props w/test</li>
+    <li>Yield additional content w/test e.g. two Hds::Button's and one Hds::Link::Standalone</li>
+    <li>Put a Percy on it!</li>
+  </ul>
+</section>
```

Comment:
@didoo @MelSumner Please let me know what you think so far, esp on this TODO list. Note: Static only for today

Definitely happy to work in any way that you prefer as you've been "owning" this repo, and this is my first real new feature task here (it's fun!)
- Please feel free to comment at will on all the things
- Can branch each

Reply:
started the new branching